sdk/storage/Azure.Storage.Common/swagger/Generator/src/generator.ts

```diff
@@ -173,32 +174,52 @@ function generateOperation(w: IndentWriter, serviceModel: IServiceModel, group:
         w.write(')')
     });
     w.scope('{', '}', () => {
-        w.write(`using (Azure.Core.Http.Request ${requestName} = ${methodName}_CreateRequest(`);
-        w.scope(() => {
-            const separateParams = IndentWriter.createFenceposter();
+        w.line(`Azure.Core.Pipeline.DiagnosticScope ${scopeName} = ${pipelineName}.Diagnostics.CreateScope("${naming.type(service.name)}.${operation.name}");`)

```

Comment:
This should probably be `"${naming.type(service.name)}.${operation.group ? operation.group + '.' : ''}${operation.name}"` so it'll look like `BlobRestClient.Service.GetProperties` and `BlobRestClient.Container.GetProperties` instead of just `BlobRestClient.GetProperties` for both.